packages/react-i18n/src/i18n.ts

```diff
@@ -314,6 +315,29 @@ export class I18n {
     return getCurrencySymbol(locale, {currency});
   }
 
+  formatName(firstName: string, lastName?: string, options?: {full?: boolean}) {
```

Comment:
Nitpick but if your list of options will grow, you could extract it into a type.

Reply:
I don't see any other potential options for this function. I'll keep it as is for now. 👍